Client/trunk/ParaEngineClient/IO/FileUtils.cpp

```diff
@@ -322,9 +322,10 @@ bool ParaEngine::CFileUtils::MoveFile(const char* src, const char* dest)
 #elif defined(USE_BOOST_FILE_API)
 	try
 	{
-		fs::path sSrc(src);
-		fs::copy_file(sSrc, fs::path(dest), fs::copy_option::overwrite_if_exists);
-		return fs::remove(sSrc);
+		boost::system::error_code err_code;
+		fs::rename(sSrc, fs::path(dest), err_code);
+		OUTPUT_LOG("info (boost-fs): moved file/directory from %s to %s result message: %s\n", src, dest, err_code.message().c_str());
```

Comment:
are you sure rename support fs::copy_option::overwrite_if_exists as in the old code？ 
if overwrite is not properly supported, it will lead to errors on some platforms as I recalled.

Reply:
这个就是为了解决在linux上移动文件出现的问题的，我当时修改的log是“now ParaIO.MoveFile works in the expected way on linux (boost)”，具体问题现象是什么我现在已经记不清楚了，如果你不想要的话我还把它移回我们自己的分支？